book/overlays.md

```diff
@@ -186,13 +186,31 @@ Sometimes, you might want to remove an overlay, but keep all the custom definiti
 
 (spam)> def eggs [] { "eggs" }
 
-(spam)> overlay remove --keep spam
+(spam)> overlay remove --keep-custom spam
 
 (zero)> eggs
 eggs
 ```
 
-The `--keep` flag does exactly that.
+The `--keep-custom` flag does exactly that.
+
+One can also keep a set of environment variables that were defined inside an overlay, but remove the rest:
+
+```
+(zero)> module spam { export def foo [] { "foo" }; export env FOO {"foo"}}
+
+(zero)> overlay add spam
+
+(spam)> overlay remove spam --keep-env [FOO]
+
+(zero)> foo
+Error: Can't run executable...
+
+(zero)> $env.FOO
+foo
+```
+
+The `--keep-env` flag does exactly that.
```

Comment:
... and I'd remove this line

Reply:
i kept the same format as the `--keep-custom` flag, but yeah sure :+1:

Reply:
should be addressed in [4df4f74](https://github.com/nushell/nushell.github.io/pull/565/commits/4df4f74eb88795e9126ae8324bbc31e676b75252) :yum: